src/__tests__/reducers/articleReducer.test.js

```diff
@@ -191,4 +208,128 @@ describe('currentUserReducer', () => {
       error: 'Dislike Article Failed',
     });
   });
+  it('should handle `SUBMIT_ARTICLE_TAG`', () => {
+    const initialState = {
+      loading: true,
+      submitting: false,
+      success: true,
+      createArticle: {
+        title: '',
+        body: '',
+        tagList: [],
+      },
+      articlesList: [],
+    };
+    const expectedState = {
+      type: articleTypes.SUBMIT_ARTICLE_TAG,
+      payload: {
+        tag: 'headset',
+      },
+    };
+    expect(reducer(initialState, expectedState)).toEqual({
+      loading: true,
+      submitting: false,
+      success: true,
+      createArticle: {
+        title: '',
+        body: '',
+        tagList: [expectedState.payload.tag],
+      },
+      articlesList: [],
+    });
+  });
+
+  it('should handle `SUBMIT_ARTICLE_TAG`', () => {
```

Comment:
why space out this test but not the rest? Your code needs order

Reply:
Thank you for mentioning @kimotho-njoki. We are going to fix it